build/ExternalConsumers.props

```diff
@@ -0,0 +1,32 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
As far as I know this is the first direct references to WPF and Avalonia XPF in Avalonia. It doesn't seem like a good idea to do this and breaks good project layering IMO. Can't the text source be pulled into the private XPF repo?

Reply:
XPF has to use all the same Avalonia nuget packages under the hood (as it is planned to have integrations with avalonia controls), which means Avalonia itself needs to include InternalsVisibleTo. There was a MSBuild hack to disable internal access checks on the consuming side, but it didn't work well.

And XPF needs internals access since it is implemented on top of some Avalonia internal low-level APIs.

Reply:
Moving this specific file to the XPF repo but still using it to build stable Avalonia is a bit problematic as well.